samples/snippets/get_evaluation_sample.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START documentai_get_evaluation]
+
+from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import NotFound
+from google.cloud import documentai_v1beta3 as documentai
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID' # Create processor before running sample
+# processor_version_id = 'YOUR_PROCESSOR_VERSION_ID'
+# evaluation_id = 'YOUR_EVALUATION_ID'
+
+
+def get_evaluation_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version_id: str,
+    evaluation_id: str,
+):
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+    opts = ClientOptions(api_endpoint=f"{location}-documentai.googleapis.com")
+
+    client = documentai.DocumentProcessorServiceClient(client_options=opts)
+
+    # The full resource name of the evaluation
+    # e.g. 'projects/project_id/locations/location/processors/processor_id/processorVersions/processor_version_id/evaluations/evaluation_id'
+    evaluation_name = client.evaluation_path(
+        project_id, location, processor_id, processor_version_id, evaluation_id
+    )
+    # Make GetEvaluation request
+    try:
```

Comment:
See previous comment about `try/catch`.

Reply:
Note: if the except is hit, you'd want to exit the process. Otherwise the next set of print statements will attempt to access members of a `None` object.